sklearn/linear_model/coordinate_descent.py

```diff
@@ -389,11 +392,10 @@ def enet_path(X, y, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
 
     # X should be normalized and fit already if function is called
     # from ElasticNet.fit
-    if pre_fit:
```

Comment:
This is true, in fact `enet_path` is always called with `check_input = False` from `ElasticNet.fit`, as necessary checks has been performed already. I once thought that using two flags would be more explicit, but merging them is indeed cleaner.